server/lib/slack.js

```diff
@@ -74,13 +74,15 @@ export default {
         targetUrl = `${targetUrl.replace(/\/+$/, '')}/slack`;
       }
 
-      return new Slack(targetUrl, {}).send(slackOptions, err => {
-        if (err) {
+      return axios
+        .post(targetUrl, slackOptions)
+        .then(() => {
+          return resolve();
+        })
+        .catch(err => {
           logger.warn(`SlackLib.postMessage failed for ${targetUrl}:`, err);
```

Comment:
Can be done in a different PR, but it would be great to do a `reportErrorToSentry` here to catch similar errors in the future:

```suggestion
          logger.warn(`SlackLib.postMessage failed for ${targetUrl}:`, err);
          reportErrorToSentry(err, { extra: { targetUrl, slackOptions } });
```